features/step_definitions/networking.rb

```diff
@@ -857,6 +857,13 @@
   raise "the networkType is not #{network_type}" unless network_operator.network_type(user: _admin) == network_type
 end
 
+Given /^the cluster has "([^"]*)" endpoint publishing strategy$/ do |ep_pub_strategy|
```

Comment:
@asood-rh probably we don't need to step since the ingress_controller class is already defined [here](https://github.com/openshift/verification-tests/blob/master/lib/openshift/ingress_controller.rb#L15)

Reply:
@pruan-rht The step obtains the endpoint publishing strategy and fails the test script if it is not what was passed in by the script. This is more than a getter method.

Reply:
@pruan-rht https://github.com/openshift/cucushift/pull/8572 PR that uses the step. Could you please approve thisone?

Thanks.

Reply:
/lgtm